atd-toolbox/data_model/python/sql_templates/audit_fields.sql

```diff
@@ -1,42 +1,3 @@
-alter table public.units_cris add column created_at timestamptz not null default now();
-alter table public.units_cris add column updated_at timestamptz not null default now();
-alter table public.units_cris add column created_by text not null default 'system';
-alter table public.units_cris add column updated_by text not null default 'system';
-alter table public.people_cris add column created_at timestamptz not null default now();
-alter table public.people_cris add column updated_at timestamptz not null default now();
-alter table public.people_cris add column created_by text not null default 'system';
-alter table public.people_cris add column updated_by text not null default 'system';
-alter table public.charges_cris add column created_at timestamptz not null default now();
-alter table public.charges_cris add column created_by text not null default 'system';
-alter table public.crashes_edits add column created_at timestamptz not null default now();
-alter table public.crashes_edits add column updated_at timestamptz not null default now();
-alter table public.crashes_edits add column created_by text not null default 'system';
-alter table public.crashes_edits add column updated_by text not null default 'system';
-alter table public.units_edits add column created_at timestamptz not null default now();
-alter table public.units_edits add column updated_at timestamptz not null default now();
-alter table public.units_edits add column created_by text not null default 'system';
-alter table public.units_edits add column updated_by text not null default 'system';
-alter table public.people_edits add column created_at timestamptz not null default now();
-alter table public.people_edits add column updated_at timestamptz not null default now();
-alter table public.people_edits add column created_by text not null default 'system';
-alter table public.people_edits add column updated_by text not null default 'system';
-alter table public.crashes add column created_at timestamptz not null default now();
-alter table public.crashes add column updated_at timestamptz not null default now();
-alter table public.crashes add column created_by text not null default 'system';
-alter table public.crashes add column updated_by text not null default 'system';
-alter table public.units add column created_at timestamptz not null default now();
-alter table public.units add column updated_at timestamptz not null default now();
-alter table public.units add column created_by text not null default 'system';
-alter table public.units add column updated_by text not null default 'system';
-alter table public.people add column created_at timestamptz not null default now();
-alter table public.people add column updated_at timestamptz not null default now();
-alter table public.people add column created_by text not null default 'system';
-alter table public.people add column updated_by text not null default 'system';
```

Comment:
you are just deleting all this in favor of having them in the up migrations for these tables instead right?

Reply:
correct!